src/api/index.tsx

```diff
@@ -15,6 +15,7 @@ type CallApiParams = {
   method?: HttpMethod;
   version?: string;
   apiState: ApiState;
+  query?: { [key: string]: string };
```

Comment:
Nit: Do you think maybe `params` would be a better name for this?

Reply:
`params` is a bit too generic to me. `query` could be generic but in this context, it is more meaningful. `queryParameters` would be too long, and `qs` or `queryString` would be a bad name for an object value.